game/customization.lua

```diff
@@ -67,7 +67,7 @@ local function filterPedModelsForPlayer(pedConfigs)
 
     for i = 1, #pedConfigs do
         local config = pedConfigs[i]
-        if (not config.jobs and not config.gangs and not config.aces) or allowedForPlayer(config, allowedAces) then
+        if (not config.jobs and not config.gangs and not config.aces and not config.citizenids) or allowedForPlayer(config, allowedAces) or (config.citizenids and listContains(config.citizenids, client.citizenid)) then
```

Comment:
All the conditions are already checked in `allowedForPlayer` function. You don't need to add the checks again here. See `filterBlacklistSettings` function on how to change this condition properly

Reply:
but `allowedForPlayer` just for aces right ? not citizenids

Reply:
No. It has checks for jobs, gangs, aces AND citizenids. Take a look at that function again

Reply:
ah i see,but i tried at pedblacklisting with citizenids not work for filtering

Reply:
The check just needs to be this:
```lua
        if (not config.jobs and not config.gangs and not config.aces and not config.citizenids) or allowedForPlayer(config, allowedAces) then
```

It was missing the config.citizenids part which is probably why it didn't work. But the rest of the condition is already checked in `allowedForPlayer` function

Reply:
![image](https://user-images.githubusercontent.com/60453612/220027414-961c31cd-360b-4c72-bdad-38b7a48b7552.png)
Still appear when i make "TEST" citizenids
![image](https://user-images.githubusercontent.com/60453612/220027703-378bb294-b937-4470-9e58-8165220a00ac.png)

Reply:
> t was missing the config.citizenids part which is probably why it didn't work. But the rest of the condition is already

ah yeah, this fixed

Reply:
can i make PR again ?

Reply:
I would suggest making changes to the existing PR. Its easier to track

Reply:
You can push more changes to your main branch in your github and they will automatically appear in this PR

Reply:
> You can push more changes to your main branch in your github and they will automatically appear in this PR

alright wait a sec